cellxgene_schema_cli/tests/test_schema_compliance.py

```diff
@@ -224,6 +224,27 @@ def test_column_presence_assay(self):
             ],
         )
 
+    def test_obs_reserved_columns_presence(self):
+        """
+        Reserved columns must NOT be used in obs
+        """
+
+        for reserved_column in self.validator.schema_def["components"]["obs"]["reserved_columns"]:
+            with self.subTest(column=reserved_column):
```

Comment:
we are using pytest for running the tests and it does not play well with subtests. I suggest using https://docs.pytest.org/en/7.3.x/how-to/parametrize.html to achieve the same behavior.

Reply:
Where does it not playing well manifest? When I run `make unit-test` everything seems fine, and we use subtests all throughout the test suite for the repo. I'm down to look into it, just wondering if we have worse test coverage than we think if subtests aren't working properly

Reply:
tracking separate issue for this https://app.zenhub.com/workspaces/single-cell-5e2a191dad828d52cc78b028/issues/gh/chanzuckerberg/single-cell-curation/633